app/views/users/show.html.erb

```diff
@@ -1 +1,18 @@
-<h1>Here is a list of a specific user</h1>
\ No newline at end of file
+<%= render "user", user: @user %>
+<div class="bio">
+  <h3>Bio<h3>
+  <p><%= @user.bio %></p>
+</div>
+<ul>
+  <% @recent_posts.each do |post| %>
+    <li>
+      <article>
+        <h2><%= post.title %></h2>
+        <p><%= post.text %></p>
+        <p>Comments: <%= post.comments_counter || 0 %></p>
+        <p>Likes: <%= post.likes_counter || 0 %></p>
+      </article>
+    </li>
+  <% end %>
+</ul>
+<button type="button" >See all posts<button>
```

Comment:
- [ ] Hello @youmari nice job so far however, the button to see all the posts and comments is not rendered correctly. I can't see my posts or comments.

![Capture web_29-3-2022_85145_localhost](https://user-images.githubusercontent.com/61562626/160573578-752d887c-8cb0-4118-8503-2af2894d69cb.jpeg)

Reply:
- [ ] This issue is not yet implemented or fixed, please fix it before you request another review.

Reply:
As suggested by the previous reviewer, the button or link to see all the posts and comments is not rendered correctly. I can't see my posts or comments.